### ISSUE TITLE
Install kubesphere for the k8s cluster  failed on the arm64 node 

### ISSUE THREAD

**Issue by Nick1211** [kubesphere/ks-installer#1847]
**Env:**

x86 master
arm64 node

Kubernetes:
root@master:~# kubectl version
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.8", GitCommit:"fd5d41537aee486160ad9b5356a9d82363273721", GitTreeState:"clean", BuildDate:"2021-02-17T12:41:51Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.8", GitCommit:"fd5d41537aee486160ad9b5356a9d82363273721", GitTreeState:"clean", BuildDate:"2021-02-17T12:33:08Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"linux/amd64"}

KubeSphere:
v3.0.0

**Step:**
1.Install Kubernetes on the k8s cluster successfully ( one x86 master and one arm64 node)

2.Install kubesphere on the k8s cluster.

**Exception:**
Install kubesphere on the k8s cluster which combined by one x86 master and one arm64 node, but arm64 node install kubesphere failed on the k8s cluster due to pulled and used the x86 docker image.

**Expect:**
pull the right arch docker image in the arm64 node

**Comment by FeynmanZhou** [comment]
Hi @Nick1211 , have you followed this [guide](https://kubesphere.com.cn/forum/d/3113-arm64-k8s-kubesphere/24) to install KubeSphere on ARM 64 nodes?

**Comment by Nick1211** [comment]
> Hi @Nick1211 , have you followed this [guide](https://kubesphere.com.cn/forum/d/3113-arm64-k8s-kubesphere/24) to install KubeSphere on ARM 64 nodes?

Yes, it worked fine when all in one for the only one arrch64 device node as the master and worker node.

But it's failed when I set one x86 server as the master, and another arrch64 device  as the node,  also I set the arch option in the cluster config.xml, the node will always pull the wrong arch docker image of the ks-installer related service.

**Comment by Nick1211** [comment]
> /help



> Hi @Nick1211 , have you followed this [guide](https://kubesphere.com.cn/forum/d/3113-arm64-k8s-kubesphere/24) to install KubeSphere on ARM 64 nodes?

Yes, it worked fine when all in one for the only one arrch64 device node as the master and worker node.

But it's failed when I set one x86 server as the master, and another arrch64 device  as the node,  also I set the arch option in the cluster config.xml, the node will always pull the wrong arch docker image of the ks-installer related service.
